airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -119,6 +150,17 @@ def __init__(
         except ValueError:
             logger.setLevel(logging.WARNING)
 
+    def _get_field(self, extra_dict, field_name):
+        prefix = 'extra__wasb__'
+        if field_name.startswith('extra_'):
```

Comment:
Same as the previous PR review comment, can we check for the exact key `extra__`